plotly/graph_reference.py

```diff
@@ -39,6 +39,8 @@
     'Histogram2d': {'object_name': 'histogram2d', 'base_type': dict},
     'Histogram2dContour': {'object_name': 'histogram2dcontour',
                            'base_type': dict},
+    'Histogram2dcontour': {'object_name': 'histogram2dcontour',
+                           'base_type': dict},
```

Comment:
this change seems odd to me, seems like something that should exist (if necessary) in #942. What's the rational behind this change?

Reply:
I think we should be able to delete `graph_reference.py` altogether (but maybe it's still used for testing?). I copied this mapping to `codegen/compatibility.py` to use when generating the deprecated classes.

Reply:
OK yeah, this seems like something we can delete. I forgot all about `graph_reference` - this is from the old days when we didn't have a plotly.s plot-schema!

@Kully - Can you chime in here? Do we still need `graph_reference.py`?